azuredeploy.parameters.json

```diff
@@ -7,6 +7,7 @@
                 "webServerType":               { "value": "apache" },
                 "dbServerType":                { "value": "mssql" },
                 "fileServerDiskCount":         { "value": 2 },
-                "fileServerDiskSize":          { "value": 32 }
+                "fileServerDiskSize":          { "value": 32 },
+                "searchType":                  { "value": "azure-search" }
```

Comment:
This should be removed (with the default change to 'non-search') so that the Azure/azure-quickstart-templates test deployment time is not extended any longer.